docs/users_guide/ocean/tasks/external_gravity_wave.md

```diff
@@ -0,0 +1,237 @@
+The default resolutions used in the task depends on the mesh type.
+
+For the `icos` mesh type, the default is 60, as determined
+by the following config option. See {ref}`dev-ocean-convergence` for more
+details.
+
+```cfg
+# config options for spherical convergence tests
+[spherical_convergence]
+
+# The base resolution for the icosahedral mesh to which the refinement
+# factors are applied
+icos_base_resolution = 60.
+```
+
+For the `qu` mesh type, it is 120 km as
+determined by the following config option:
+
+```cfg
+# config options for spherical convergence tests
+[spherical_convergence]
+
+# The base resolution for the quasi-uniform mesh to which the refinement
+# factors are applied
+qu_base_resolution = 120.
+
+# a list of quasi-uniform mesh resolutions (km) to test
+qu_refinement_factors = 0.5, 0.75, 1., 1.25, 1.5, 1.75, 2.
+```
+
+To alter the resolutions used in the convergence tasks, you will need to create
+your own config file (or add a `spherical_convergence` section to a config file
+if you're already using one).  If you specify a different resolution
+before setting up `external_graivty_wave`, steps will be generated with
+the requested resolution (if you alter `icos_resolutions` or `qu_resolutions`
+in the task's config file in the work directory, nothing will happen).
+For `icos` meshes, make sure you use a resolution close to those listed in
+{ref}`dev-spherical-meshes`. Each resolution will be rounded to the nearest
+allowed icosahedral resolution.
+
+The `base_mesh` steps are shared with other tasks so they are not housed in
+the `external_gravity_wave` work directory. Instead, they are in work
+directories like:
+
+```
+ocean/spherical/icos/base_mesh/60km
+ocean/spherical/qu/base_mesh/60km
+```
+
+(ocean-external-gravity-wave-vertical)=
+## vertical grid
+
+For this task, only a single vertical level may be used. The bottom depth
+is constant and the results should be insensitive to the choice of
+`bottom_depth`.
+
+```cfg
+# Options related to the vertical grid
+[vertical_grid]
+
+# the type of vertical grid
+grid_type = uniform
+
+# Number of vertical levels
+vert_levels = 1
+
+# Depth of the bottom of the ocean
+bottom_depth = 1000.0
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-level
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# The minimum fraction of a layer for partial cells
+min_pc_fraction = 0.1
+```
+
+## initial conditions
+
+The fluid velocity is initialized to zero, and the layer thickness is given as
+a simple Gaussian bump
+
+$$
+h = e^{-100 \left( \left(\phi - \phi_0\right)^2 + \left(\theta - \theta_0\right)^2 \right)}
+$$
+
+where $\phi$ is latitude, $\theta$ is longitude, $\phi_0 = 0$, and
+$\theta_0 = \pi / 2$. Here, $\phi_0$ and $\theta_0$ can be set by `lat_center`
+and `lon_center` as config options.
+
+## forcing
+
+This case run with all momentum equation terms disabled save the pressure
+gradient term.
```

Comment:
```suggestion
N/A
```
We usually only specify external forcing or internal forcing like salinity restoring in this section.